pkg/proxy/apis/kubeproxyconfig/types.go

```diff
@@ -97,13 +97,13 @@ type KubeProxyConntrackConfiguration struct {
 type KubeProxyConfiguration struct {
 	metav1.TypeMeta
 
-	// featureGates is a comma-separated list of key=value pairs that control
-	// which alpha/beta features are enabled.
-	//
-	// TODO this really should be a map but that requires refactoring all
+	// TODO FeatureGates really should be a map but that requires refactoring all
```

Comment:
Fixing this should be a prereq for moving to beta IMO. This is already a map[string]bool for the kubelet
cc @xiangpengzhao @mtaufen

Reply:
Agreed. PR https://github.com/kubernetes/kubernetes/pull/57962 sent.